Cargo.lock

```diff
@@ -85,18 +85,33 @@ checksum = "739f4a8db6605981345c5654f3a85b056ce52f37a39d34da03f25bf2151ea16e"
 
 [[package]]
 name = "aho-corasick"
-version = "0.7.18"
+version = "1.0.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "1e37cfd5e7657ada45f742d6e99ca5788580b5c529dc78faf11ece6dc702656f"
+checksum = "43f6cb1bf222025340178f382c426f13757b2960e89779dfcb319c32542a5a41"
 dependencies = [
  "memchr",
 ]
 
+[[package]]
```

Comment:
Why are these new dependencies?  Please remove them -- there's no reason to depend on Android-specific things because we don't support Android.

Reply:
I think you can just remove `chrono`.  It's only needed by `stacks-node`; the `chrono` dependency really ought to be present only in `testnet/stacks-node/Cargo.toml`.